release.yaml

```diff
@@ -99,16 +99,19 @@ rancher-logging-crd:
 rancher-monitoring:
   - 102.0.2+up40.1.2
   - 103.0.0+up45.31.1
+  - 103.0.1+up45.31.1
 rancher-monitoring-crd:
   - 102.0.2+up40.1.2
   - 103.0.0+up45.31.1
+  - 103.0.1+up45.31.1
 rancher-project-monitoring:
   - 0.3.0+up0.3.3
 rancher-provisioning-capi:
   - 100.0.0+up0.0.1
   - 103.0.0+up0.0.1
 rancher-pushprox:
   - 103.0.0
```

Comment:
Can we remove 103.0.0 version of pushprox ? since we are adding 103.0.1 and 103.0.0 is not yet released.

Reply:
I'm open to removing `103.0.0` but I'd rather keep the distinction that `103.0.0` was brought in for the rancher `2.8` release and this was added on top.

If we don't want to have both `103.0.0` and `103.0.1` I'd rather stick this into `103.0.0` instead.

Reply:
yes, sticking to 103.0.0 makes sense to me. Thanks :)

Reply:
ok cool, I'll update those versions